src/Microsoft.Health.Fhir.Liquid.Converter/Models/DataType.cs

```diff
@@ -9,5 +9,6 @@ public enum DataType
     {
         Hl7v2,
         Ccda,
+        Json,
```

Comment:
Shall we add comments for all data types? Looks like we are going to have a long list of data types in the future.

Reply:
Thanks, I've added comments for this class to indicate that this is the input data types.